.github/workflows/ci.yml

```
@@ -0,0 +1,159 @@
+        PYTHONPATH: src
+
+  release:
+    runs-on: ubuntu-latest
+    needs: [test, integration-tests]
+    if: github.event_name == 'push' && github.ref == 'refs/heads/main'
+
+    steps:
+    - uses: actions/checkout@v4
+
+    - name: Set up Python 3.13
+      uses: actions/setup-python@v4
```

actionlint (1.7.7)

the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)